docs/eps/week1.md

```diff
@@ -1,18 +1,19 @@
 # Study Group Lecture 1 | Protocol Intro
 
-The first day of the study group is dedicated to a general introduction to the protocol and R&D ecosystem. 
+The first day of the study group is dedicated to a general introduction to the protocol and R&D ecosystem.
 
 ## Recording
 
-Watch *Ethereum Protocol 101* talk by Mario Havel [on StreamEth](https://streameth.org/watch?event=&session=65d77e4f437a5c85775fef9d). [Slides](https://github.com/eth-protocol-fellows/protocol-studies/tree/main/docs/eps/presentations/week1_protocol_intro.pdf) 
+Watch _Ethereum Protocol 101_ talk by Mario Havel [on StreamEth](https://streameth.org/watch?event=&session=65d77e4f437a5c85775fef9d). [Slides](https://github.com/eth-protocol-fellows/protocol-studies/tree/main/docs/eps/presentations/week1_protocol_intro.pdf)
```

Comment:
This PR touches a lot of other stuff, in many places it changes bold to italic, that should be reverted